README.md

```diff
@@ -90,7 +90,7 @@ Tempest was built by a team of Stanford PhDs---[Peter Lofgren](@plofgren) (lead
    directory outside docker, so you can upgrade tempest without losing your data.  To
    use Tempest DB, install docker on your machine, then run
    
-   `docker run -t -i -v $YOUR_DATA_DIR:/data -v ~/:/mnt/home/ -p 127.0.0.1:10001:10001 teapotco/tempestdb:0.12.3 bash`
+   `docker run -t -i -v $YOUR_DATA_DIR:/data -v ~/:/mnt/home/ -p 127.0.0.1:10001:10001 teapotco/tempestdb:0.13.0 bash`
```

Comment:
I agree; I'll make a tag called `latest` (a standard docker convention, like `master` in git) and use that.